pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1085,6 +1088,39 @@ func (h *HTTPArtifact) HasLocation() bool {
 	return h != nil && h.URL != ""
 }
 
+// OSSBucket contains the access information required for interfacing with an OSS bucket
+type OSSBucket struct {
+	// Endpoint is the hostname of the bucket endpoint
+	Endpoint string `json:"endpoint"`
+
+	// Bucket is the name of the bucket
+	Bucket string `json:"bucket"`
+
+	// AccessKeySecret is the secret selector to the bucket's access key
+	AccessKeySecret apiv1.SecretKeySelector `json:"accessKeySecret"`
```

Comment:
`AccessKeyID`? According to https://github.com/aliyun/aliyun-oss-python-sdk#getting-started

Reply:
thanks for your comments, it is updated.